source/classes.tex

```diff
@@ -2264,11 +2264,15 @@
 \tcode{virtual}\iref{class.virtual}
 or pure
 \tcode{virtual}\iref{class.abstract}.
+\begin{note}
 If the destructor of a class is virtual and
 any objects of that class or any derived class are created in the program,
-the destructor shall be defined.
-If a class has a base class with a virtual destructor, its  destructor
-(whether user- or implicitly-declared) is virtual.
+the destructor must be defined; see~\ref{basic.def.odr}.
+\end{note}
+\begin{note}
+If a class has a base class with a virtual destructor, its destructor
```

Comment:
While we're in that area, the "its" has an ambiguous antecedent (it obviously refers to "a virtual destructor").
Suggestion: "If a class C has a base class with a virtual destructor, the destructor of C ... is virtual (xref)."